sycl/doc/extensions/proposed/sycl_ext_intel_fpga_datapath.asciidoc

```diff
@@ -212,24 +264,39 @@ Returns a reference to the underlying `T`.
 
 Below are some non-obvious interactions between `fpga_datapath` and other SYCL extensions. 
 
-* The `fpga_datapath` and `fpga_mem` are tempate classes that have conflicting requirements on how a variable should be implemented, on or off the datapath. In situations where an object of one of these classes is templated on another, the most specific template will used by the compiler, and the rest will be ignored.
+* The `fpga_datapath` and `fpga_mem` are tempate classes that have conflicting requirements on how a variable should be implemented, on or off the datapath. In situations where an object of one of these classes is templated on another, the less specific template will used by the compiler, and the rest will be ignored.
 
 [source,c++]
 ----
 namespace intelexp = sycl::ext::intel::experimental;
 
-// 10 integers will be implemented on the datapath
-// `fpga_mem` is ignored.
+// 10 integers will be implemented off the datapath, in memory
+// `fpga_datapath` is ignored.
 intelexp::fpga_mem<intelexp::fpga_datapath<int[10]>> dm1;
 
+----
+
+* `fpga_mem` and `fpga_datapath` objects can be member variables of a struct. If the struct type itself is used as a template argument to `fpga_mem` or `fpga_datapath` any hints provided from the member variables are ignored. 
```

Comment:
Do you really need to say this?  It seems redundant given the sentence above about the less specific template winning.

Reply:
After thinking more about it, I see what you were going for here but I still don't think it's complete as you only cover one layer of indirection.  For example, I don't think either sentence covers this case:

```
struct one {
  fpga_mem<int[10]> a;
};

struct two {
  one b;
};
  
fpga_datapath<two> foo;
```

In this case I believe you want the annotation on foo.b.a to be dropped but neither of the sentences guarantees that.